SPECS/curl/curl.spec

```diff
@@ -47,7 +47,7 @@ This package contains minimal set of shared curl libraries.
     --with-ssl \
     --with-gssapi \
     --with-libssh2 \
-    --with-ca-bundle=/etc/pki/tls/certs/ca-bundle.crt \
```

Comment:
Shouldn't we add a dependency on ca-certificates?

Reply:
Same for other SPECs

Reply:
No, this is just the default path where the bundle should be placed, if you want to use it. It's not mandatory for the bundle to actually be there. You can also pick a different path using env variables or by calling `curl` or `openssl` APIs yourself.